integration-tests/features/importing_coverage.feature

```diff
@@ -36,71 +36,71 @@ Feature: Importing coverage data
-          -Dsonar.cxx.coverage.itReportPath=it-coverage.xml
-          -Dsonar.cxx.coverage.overallReportPath=overall-coverage.xml
-          -Dsonar.cxx.coverage.forceZeroCoverage=True
-          """
+#      WHEN I run sonar-scanner with following options:
+#          """
+#          -Dsonar.cxx.coverage.reportPath=ut-coverage.xml
+#          -Dsonar.cxx.coverage.itReportPath=it-coverage.xml
+#          -Dsonar.cxx.coverage.overallReportPath=overall-coverage.xml
+#          -Dsonar.cxx.coverage.forceZeroCoverage=True
+#          """
 
-      THEN the analysis finishes successfully
-          AND the analysis in server has completed      
-          AND the analysis log contains no error/warning messages except those matching:
-              """
-              .*WARN.*Unable to get a valid mac address, will use a dummy address
-              .*WARN.*cannot find the sources for '#include <iostream>'
-              """
-          AND the following metrics have following values:
-              | metric                  | value |
-              | coverage                | 8.5   |
-              | line_coverage           | 5.5   |
-              | branch_coverage         | 50    |
-              | it_coverage             | 20.3  |
-              | it_line_coverage        | 14.8  |
-              | it_branch_coverage      | 50    |
-              | overall_coverage        | 28.1  |
-              | overall_line_coverage   | 22    |
-              | overall_branch_coverage | 50    |
+#      THEN the analysis finishes successfully
+#          AND the analysis in server has completed      
+#          AND the analysis log contains no error/warning messages except those matching:
+#              """
+#              .*WARN.*Unable to get a valid mac address, will use a dummy address
+#              .*WARN.*cannot find the sources for '#include <iostream>'
+#              """
+#          AND the following metrics have following values:
+#              | metric                  | value |
+#              | coverage                | 8.5   |
+#              | line_coverage           | 5.5   |
+#              | branch_coverage         | 50    |
+#              | it_coverage             | 20.3  |
+#              | it_line_coverage        | 14.8  |
+#              | it_branch_coverage      | 50    |
+#              | overall_coverage        | 28.1  |
+#              | overall_line_coverage   | 22    |
+#              | overall_branch_coverage | 50    |
 
 
-  Scenario: Zeroing coverage measures without importing reports
+#  Scenario: Zeroing coverage measures without importing reports
 
-      If we don't pass coverage reports *and* request zeroing untouched
-      files at the same time, all coverage measures, except the branch
-      ones, should be 'zero'. The branch coverage measures remain 'None',
-      since its currently ignored by the 'force zero...'
-      implementation
+#      If we don't pass coverage reports *and* request zeroing untouched
```

Comment:
manipulating measures in batch side is no longer supported, if needed we must move the zeroing to compute engine side. I try putting this in a line visitor, so that all lines are zeroed before even reaching the coverage sensor, however this resulted in duplicated measures exceptions.  so this cant be done from batch side.

to be decided if we try to implement this in server side, however i have no idea atm how to do it. #909